### PR TITLE
Add Mellanox CX-4 and CX-6 ib NIC to supported nic list

### PR DESCRIPTION
**Pull request by pliurh**

### CODE REVIEW

File: pkg/apis/sriovnetwork/v1/helper.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ var NicIdMap = []string{
 	"8086 158b 154c",
 	"15b3 1015 1016",
 	"15b3 1017 1018",
+	"15b3 101b 101c",
```
Comment (zshi-redhat) [inline pkg/apis/sriovnetwork/v1/helper.go:38]: CX-4: `15b3 1013 1014`

Comment (pliurh) [reply pkg/apis/sriovnetwork/v1/helper.go:38]: Done

**Pull request merged**